CMP1903_A1_2324/Die.cs

```diff
@@ -8,16 +8,21 @@ namespace CMP1903_A1_2324
 {
     internal class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
+        //By making the Random instance static, it ensures that all rolls of the dice will continue along the same seed.
+        public static Random RNG = new Random();
 
-        //Property
+        //Provide a public way to get the current value of the dice. Set is private, as the dice will not need to be set outside of the roll function.
+        public int CurrentValue { get; private set; }
 
+        //This method returns a random number between 1 and 6.
+        public int Roll()
+        {
+            int rolledValue = RNG.Next(1, 7); //Using the static Random instance, generate a random integer between 1 and 6.
 
-        //Method
-
+            //Set the current value of the dice to the random value, and return it.
+            CurrentValue = rolledValue;
+            return rolledValue;
+        }
 
```

Comment:
Empty line could be removed, however I understand why it was left there to separate the end of the Roll method.  Seeing how this was not done in the other classes however, I would recommend removing this.